### ISSUE TITLE
alias: Property alias is not allowed

### ISSUE THREAD

**Issue by oneseventhree** [keesschollaart81/vscode-home-assistant#2371]
Same issue as in https://github.com/keesschollaart81/vscode-home-assistant/issues/2293: 
![image](https://user-images.githubusercontent.com/15043282/205550794-b2b0acfc-ad0e-4f8d-99f2-425b1c088909.png)

```YAML
- id: '1651658221136'
  alias: Home Lights - Scene Schedule
  description: 'Option 1 - Relax 10 minutes after Sunset

    Option 2 - Circadian at 9pm

    Option 3 - Bright after Sleep and 6am'
  trigger:
  - platform: state
    entity_id:
    - binary_sensor.outside_darkness
    to: 'on'
    alias: RELAX - Sun Elevation
  - platform: time
    at: '21:00:00'
    id: 9pm - Bedtime Lights
    alias: CIRCADIAN - 9pm
  - platform: state
    entity_id: input_select.home_status
    to: Sleep
    for:
      hours: 1
      minutes: 0
      seconds: 0
    id: Sleep
    alias: BRIGHT - Home Status Sleep for 1 hour
  - platform: time
    at: 06:00:00
    id: 6am - Backup
    alias: BRIGHT - 6am
```

Running:
![image](https://user-images.githubusercontent.com/15043282/205550968-357330cf-bd00-4e5a-8ada-1704a7129735.png)

**Comment by TNTLarsn** [comment]
It's already been fixed: https://github.com/keesschollaart81/vscode-home-assistant/pull/2277
But not released yet

**Comment by frenck** [comment]
fixed in #2277
